ui-participant/src/Navbar.tsx

```diff
@@ -52,6 +53,8 @@ export default function Navbar(props: NavbarProps) {
   const navLinks = localContent.navbarItems
 
   async function updatePreferredLanguage(selectedLanguage: string) {
+    //track the language change
+    mixpanel.track('languageUpdated', { language: selectedLanguage })
```

Comment:
how about adding `source: 'navbar'` so we'll know where the language was changed?